### DISCUSSION TITLE
Running Tests without V8

### DISCUSSION THREAD

**Discussion by briell99** [ClearFoundry/ClearScript#276] [Q&A]
How can we run ClearScriptTests with ClearScript.NoV8?

**Comment by ClearScriptLib** [comment]
Hi @briell99,

ClearScript's test projects have always combined V8-dependent and V8-independent code – sometimes in the same test. That's why the `ClearScript.NoV8` solution has never included those projects.

If you prefer not to build V8, the only thing we can suggest is that you start with `ClearScript.NoV8`, add one or more of the existing test projects, and remove any test code that requires V8.

As an alternative, if you're just trying to reproduce a bug, consider creating a simple command-line application instead of writing a new test. That way you can add ClearScript via NuGet instead of building it. If you can isolate the bug, we'll be happy to investigate it and convert your application into a new coded test.

Good luck!
